File: beginner/1012.ts
import * as fs from 'fs';
import * as path from 'path';

const myPath = path.join(__dirname, 'file.txt');
const input = fs.readFileSync(myPath, 'utf8');

const valores = input.split(' ');

const areaTrianguloRetanglo = (Number(valores[0]) * Number(valores[2])) / 2;
const areaCirculo = 3.14159 * Math.pow(Number(valores[2]), 2);
const areaTrapezio = (Number(valores[0]) + Number(valores[1])) * Number(valores[2]) / 2;
const areaQuadrado = Math.pow(Number(valores[1]), 2);
const areaRetangulo = Number(valores[0]) * Number(valores[1]);

console.log('TRIANGULO: ' + areaTrianguloRetanglo.toFixed(3));
console.log('CIRCULO: ' + areaCirculo.toFixed(3));
console.log('TRAPEZIO: ' + areaTrapezio.toFixed(3));
console.log('QUADRADO: ' + areaQuadrado.toFixed(3));
console.log('RETANGULO: ' + areaRetangulo.toFixed(3));

File: beginner/1012.c
#include <stdio.h>

int main() {
  double PI = 3.14159;
  double a, b, c;
  scanf("%lf%lf%lf", &a, &b, &c);
  double triangleRectangleArea = (a * c) / 2.0;
  double circleArea = PI * (c * c);
  double trapezeArea = (a + b) / 2.0 * c;
  double squareArea = b * b;
  double rectangleArea = a * b;
  printf("TRIANGULO: %.3f\n", triangleRectangleArea);
  printf("CIRCULO: %.3f\n", circleArea);
  printf("TRAPEZIO: %.3f\n", trapezeArea);
  printf("QUADRADO: %.3f\n", squareArea);
  printf("RETANGULO: %.3f\n", rectangleArea);
  return 0;
}
